tests/test_lldp.py

```diff
@@ -14,6 +14,7 @@
 from ax_interface.mib import MIBTable
 from sonic_ax_impl.mibs import ieee802_1ab
 
+import tests.mock_tables.dbconnector
```

Comment:
>import [](start = 0, length = 6)

Move 'import' to after line ```from unittest import TestCase```, so it will not be override by later direct/indirect import.